File: Difficulty: Easy/Second Largest/second-largest.py
#User function Template for python3
class Solution:
    def getSecondLargest(self, arr):
        temp = list(set(arr))
        temp.sort()
        if len(temp) > 1:
            return temp[-2]
        return -1
        # Code Here


#{ 
 # Driver Code Starts
# Initial Template for Python 3

if __name__ == "__main__":
    t = int(input())
    for _ in range(t):
        arr = list(map(int, input().split()))
        ob = Solution()
        ans = ob.getSecondLargest(arr)
        print(ans)
        print("~")
# } Driver Code Ends

File: Difficulty: Easy/Second Largest/second-largest.cpp
class Solution {
  public:
    int getSecondLargest(vector<int> &arr) {
        // code here
        int lar=-1, res=-1;
        for(int i=0;i<arr.size();i++){
            if(arr[i] > lar){
                res=lar;
                lar=arr[i];
            } else if(arr[i] < lar && arr[i]> res){
                res=arr[i];
            }
        }
        return res;
    }
};